scaleway/resource_rdb_database.go

```diff
@@ -0,0 +1,160 @@
+package scaleway
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func resourceScalewayRdbDatabase() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceScalewayRdbDatabaseCreate,
+		ReadContext:   resourceScalewayRdbDatabaseRead,
+		DeleteContext: resourceScalewayRdbDatabaseDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(defaultRdbInstanceTimeout),
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validationUUIDorUUIDWithLocality(),
+				Description:  "Instance on which the database is created",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "Database name",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"managed": {
+				Type:        schema.TypeBool,
+				Description: "Whether or notre the database is managed",
+				Computed:    true,
+			},
+			"owner": {
+				Type:        schema.TypeString,
+				Description: "User that own the database",
+				Computed:    true,
+			},
+			"size": {
+				Type:        schema.TypeString,
+				Description: "Size of the database",
+				Computed:    true,
+			},
+			// Common
+			"region": regionSchema(),
```

Comment:
Do we usually include it for nested resources ?

Reply:
I would say yes but I don't have the rational : 
```
$ ack "(region|zone)Schema" | grep "scaleway/resource_" -c
25
$ find scaleway -name "resource*" -ls |grep -v _test.go -c
29
```
25 over 29 of our resource include a region or a zone attribute.

https://github.com/scaleway/terraform-provider-scaleway/search?q=regionSchema
https://github.com/scaleway/terraform-provider-scaleway/search?q=zoneSchema